ew/cmd/item/itemcmds.py

```diff
@@ -2258,7 +2258,7 @@ async def unearth(cmd):
 async def huff(cmd):
     user_data = EwUser(member = cmd.message.author)
     status = user_data.getStatusEffects()
-    item_sought = bknd_item.find_item(item_search=ewcfg.weapon_id_thinnerbomb, id_user=cmd.message.author.id, id_server=cmd.guild.id, item_type_filter=ewcfg.it_weapon)
+    item_sought = bknd_item.find_item(item_search=ewcfg.weapon_id_thinnerbomb, id_user=cmd.message.author.id, id_server=cmd.guild.id, item_type_filter=ewcfg.it_weapon) or bknd_item.find_item(item_search=ewcfg.weapon_id_spraycan, id_user=cmd.message.author.id, id_server=cmd.guild.id, item_type_filter=ewcfg.it_weapon)
```

Comment:
This could be made cleaner and more functional by instead having the player specify an item to search. It'd allow the player to choose to huff either a spraycan or a thinnerbomb, even if both are in the inventory.